Arduplane/Parachute: Little fixings

Davidsastresas opened the pull request:
The singleton panic message was saying "rally must be singleton" instead of "parachute must be singleton".

Also, in the function parachute_manual_release() we were setting landing gear position when it gets set before, when calling parachute_release(), so we were calling it two times when the release was manual.

peterbarker commented:
First commit message needs help; should be `AP_Parachute:` rather than `parachute.h`

peterbarker commented:
I don't think this is NFC.

There are various checks at the top of `parachute_release` which may inhibit it from deploying the landing gear.  So if someone triggers a manual release ATM the gear will deploy, after this it may not.

I'm not saying that this isn't a good patch, just that it's not NFC.

Davidsastresas commented:
Thank you for your comments Peter.

I changed the commit message of the first commit as per your request.

Regarding the second commit you are right, I have dropped it. 

Thanks!

peterbarker commented:
Merged, thank!
